app/src/main/assets/shared_error_style.css

```diff
@@ -64,17 +64,32 @@ p {
 
 button {
   display: block;
-  height: 36px;
+  height: 48px;
   width: 100%;
-  border-radius: 5px;
   border: 0;
+  padding: 6px 0px;
```

Comment:
We're combining padding and margin. We should probably just pick one so the button padding doesn't increase significantly. I think its alright to only use the padding value and remove the margin.